.github/scripts/toolchain/group-gcc-test-failures.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+source `dirname ${BASH_SOURCE[0]}`/../config.sh
+
+TAG=$1
+
+python3 `dirname ${BASH_SOURCE[0]}`/group-gcc-test-failures.py \
+    --dir $ARTIFACT_PATH/gcc-tests-$TAG >> \
+    $ARTIFACT_PATH/gcc-tests-$TAG/grouped-test-failures.md
+
+cat $ARTIFACT_PATH/gcc-tests-$TAG/grouped-test-failures.md |
+`dirname ${BASH_SOURCE[0]}`/extract-most-frequent-failures.sh >> $GITHUB_STEP_SUMMARY
```

Comment:
I'd not add dependency of this script on running in GitHub Actions environment. Or at least I'd add check if this is the case and output to `$GITHUB_STEP_SUMMARY` only then.